src/sigal/gallery.py

```diff
@@ -264,7 +264,7 @@ def exif(self):
         datetime_format = self.settings["datetime_format"]
         return (
             get_exif_tags(self.raw_exif, datetime_format=datetime_format)
-            if self.raw_exif and self.src_ext in (".jpg", ".jpeg")
+            if self.raw_exif and self.src_ext in (".jpg", ".jpeg", ".heic")
```

Comment:
The extension list here and below should also include ".heif" I guess ?
Also, those extensions should be recognized only if pillow-heif is available, so might be better to store this list somewhere where it can be imported (in the `image` module ?) and reused.

Reply:
I'm not sure about `.heif` files. As I understand it, this is more of a container format and may contain different image-formats.
I guess `pillow-heif` should be able to handle them all, but I have no way to test that yet.

I would actually prefer it if the program errors out if it encounters `.heic` images without `pillow-heif` installed, instead of just silently ignoring and skipping those pictures.
I mean, they are in fact Images - no matter if supported or not - and the user probably intended for them to be included in the gallery.